tempel.el

```diff
@@ -348,10 +363,21 @@ PROMPT is the optional prompt/default value."
 
 (defun tempel--save ()
   "Save template file buffer."
-  (when-let (buf (get-file-buffer tempel-file))
-    (with-current-buffer buf
-      (when (and (buffer-modified-p) (y-or-n-p (format "Save file %s? " tempel-file)))
-        (save-buffer buf)))))
+  (let (modified)
+    (dolist (file (tempel--expand-locations tempel-paths))
+      (when-let (buff (get-file-buffer tempel-file))
+  	(push file modified)))
+    (when (or (and (= (length modified) 1)
```

Comment:
A tiny nitpick: Usually I use `(not (cdr modified))` if I want to check for single elements. This is O(1) instead of O(n) :)

Furthermore it seems you miss the check for `(not modified)`? And what is the `(get-file-buffer tempel-file)` doing here? You should check for each file if a buffer exists and then check if this buffer is modified. Compare the existing code.